packages/select/custom-resources/AvPayerSelect.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import AvApi from '@availity/api-axios';
+
+import { ResourceSelect } from '..';
+
+const extendedPayersApi = new AvApi({ path: '/api/internal', name: 'extended-payers'});
+
+extendedPayersApi.all = parameters => { extendedPayersApi.query({params: { state: parameters.region, transactionType: parameters.tranTypeCode }, headers: {'X-Availity-Customer-ID': parameters.customerId}})};
```

Comment:
Two things about this function. Is it used? since `pageAll` is false, I think this won't be called. Also, looks like this function doesn't return anything if it is being called

Reply:
When it was set to pageAll: true it was calling the endpoint constantly and not displaying any of the returning payers. When I set it to false it would call the endpoint once and display the results.